docs/dml-of-carbondata.md

```diff
@@ -266,7 +281,7 @@ CarbonData DML statements are documented here,which includes:
       numPartitions = total size of input data / splitSize
     ```
     The default value is 3, and the range is [1, 300].
- 
+
```

Comment:
please revert all these changes below in this PR. Space related changes

Reply:
OK, I will do it.